src/python/grpcio/grpc/_cython/_cygrpc/aio/completion_queue.pxd.pxi

```diff
@@ -12,18 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+cdef extern from "<queue>" namespace "std" nogil:
```

Comment:
I think you can just do [`from libcpp.queue cimport queue`.](https://github.com/cython/cython/blob/master/Cython/Includes/libcpp/queue.pxd)

Reply:
If we use "cimport", we will have a build failure on some platform due to C++ exception handling. So, I avoided [`except +`](https://github.com/cython/cython/blob/master/Cython/Includes/libcpp/queue.pxd#L3). Comments added:

```
# NOTE(lidiz) Unfortunately, we can't use "cimport" here because Cython
# links it with exception handling. It introduces new dependencies.
```